rslib/src/storage/card/get_costs_for_retention.sql

```diff
@@ -0,0 +1,44 @@
+WITH searched_revlogs AS (
+  SELECT *
+  FROM revlog
+  ORDER BY id DESC -- Use the last 10_000 reviews
+  LIMIT 10000
+), average_pass AS (
+  SELECT AVG(time)
+  FROM searched_revlogs
+  WHERE ease > 1
+),
+lapse_count AS (
+  SELECT COUNT(time) AS lapse_count
+  FROM searched_revlogs
+  WHERE ease = 1
+    AND type = 1
+),
+fail_sum AS (
+  SELECT SUM(time) AS total_fail_time
+  FROM searched_revlogs
+  WHERE ease = 1
```

Comment:
```suggestion
  WHERE (ease = 1 AND type = 1)
```

The previous code would include the answer time of failures during Learn state too.